### DISCUSSION TITLE
Welcome to reoptimization-algorithms Discussions!

### DISCUSSION THREAD

**Discussion by mek97** [mek97/reoptimization-algorithms#57] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kabinesh1403** [comment]
hye im a student and i dont know how to solve a question.it will be helpful if one of you help me
